src/System.Collections.Specialized/tests/Resources/System.Collections.Specialized.Tests.rd.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Collections.Specialized.Tests">
+    <Type Name="System.Collections.Specialized.ListDictionary" Dynamic="Required All" />
```

Comment:
You should be able to use BinaryFormatter degree instead of Dynamic for this (it should propagate to members).